pandas/io/common.py

```diff
@@ -1,21 +1,24 @@
 """Common IO api utilities"""
 
 import codecs
-from contextlib import closing, contextmanager
 import csv
+from http.client import HTTPException  # noqa
 import mmap
 import os
+from urllib.error import URLError  # noqa
```

Comment:
Do you know why these need noqa now but not previously?

Reply:
Probably because `flake8` got confused by the conditional importing we had before (notice how the `noqa` is on the last import of `URLError` before this change).